c++ training/W2S2/Spotify/main.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include "Spotify.h";
+
+int main() {
+	Spotify spot = Spotify();
```

Comment:
There is no need to assign any value to spot. Writing `Spotify spot;` will call the default constructor. Also, spot is an ambiguous name, do you really need to abbreviate spotify?